foerder.html

```diff
@@ -226,12 +226,18 @@
 				formDict[pair[0]] = pair[1];
 			}
 
-			var jsonMsg = JSON.stringify(formDict);
```

Comment:
This would make it rather hard to validate/sanitize the user input. It feels a bit funny to offer a bunch of hackers a qr-code worth of direct keyboard input into dbase?